security/pkg/pki/util/san.go

```diff
@@ -119,7 +119,7 @@ func BuildSANExtension(identites []Identity) (*pkix.Extension, error) {
 		return nil, fmt.Errorf("failed to marshal the raw values for SAN field (err: %s)", err)
 	}
 
-	return &pkix.Extension{Id: oidSubjectAlternativeName, Value: bs}, nil
+	return &pkix.Extension{Id: oidSubjectAlternativeName, Critical: true, Value: bs}, nil
```

Comment:
Add the same comment to be consistent with https://github.com/istio/istio/pull/12838/files?

Reply:
Done. Thanks.